torch/onnx/symbolic_opset9.py

```diff
@@ -2127,6 +2127,11 @@ def nonzero(g, input):
     return t(g, g.op('NonZero', input))
 
 
+# Emitted from `torch.nonzero(x, as_tuple=True)`
```

Comment:
It is mentioned `torch.nonzero` emit `nonzero_numpy`, then which torch operator emits `nonzero` above?

Reply:
added comment for `nonzero` above, emitted from `torch.nonzero(x, as_tuple=False)`. This one is from `torch.nonzero(x, as_tuple=True)`.